cypress/e2e/files/share-file.cy.ts

```diff
@@ -48,5 +48,11 @@ describe("Share File Tests", () => {
     cy.get("#share-file").click({ force: true });
     // Click the "Share" button to share the element
     cy.contains("The file is already shared with the user");
+
+    // Click the "Un-Share" button to rename the folder
+    cy.get("button").contains("Un-share").should("exist");
+    cy.get("button:contains('Un-share')").click();
+    cy.contains("File unshared successfully");
+    cy.get("ul").should("not.contain", "UserToRemove");
```

Comment:
Clearly "UserToRemove" wouldn't appear in the UI, that string isn't being used anywhere in the test, I think you should use the `username` variable. 

Please, check that the user is visible after line 50 to be sure that the selector is correct.